modules/flowable-bpmn-model/src/main/java/org/flowable/bpmn/model/GraphicInfo.java

```diff
@@ -128,7 +128,7 @@ public boolean equals(GraphicInfo ginfo) {
     	}
 
     	// only check for elements that support this value
-    	if (null != this.getExpanded() && null != ginfo.getExpanded() && this.getExpanded() != ginfo.getExpanded()) {
+    	if (null != this.getExpanded() && null != ginfo.getExpanded() && !this.getExpanded().equals(ginfo.getExpanded())) {
```

Comment:
Hi @qiudaoke , can you remove this change? Because the null checks done first ensure that a safe boolean != check can be done.

Reply:
ok, i will do it